README.md

```diff
@@ -1257,7 +1254,6 @@ See also [Static site generators](#static-site-generators)
 ### Localization
 
 - [Accent](https://www.accent.reviews/) - Open-source, self-hosted, developer-oriented translation tool. ([Source Code](https://github.com/mirego/accent)) `BSD-3-Clause` `Elixir`
-- [Parrot](http://parrot-translate.github.io/parrot.github.io/) - Self-hosted Localization Management Platform built with Go and Angular 2. ([Source Code](https://github.com/parrot-translate/parrot)) `MIT` `Go`
```

Comment:
Archived by the developers. Recommends traduora as an alternative. Most obvious case.

:+1:

Reply:
@evaryont thanks for the reviews and explanations, this is exactly it.